src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Integration/Extensions/IntegrationDependencyInjectionExtensions.cs

```diff
@@ -29,11 +30,41 @@ public static WebApplicationBuilder AddAccessManagementIntegration(this WebAppli
         builder.Services.AddHttpClient<IPartiesClient, PartiesClient>();
         builder.Services.AddHttpClient<IProfileClient, ProfileClient>();
         builder.Services.AddHttpClient<IAccessListsAuthorizationClient, AccessListAuthorizationClient>();
-        builder.Services.AddHttpClient<IAltinnRolesClient, AltinnRolesClient>();
         builder.Services.AddHttpClient<IAltinn2RightsClient, Altinn2RightsClient>();
         builder.Services.AddHttpClient<IAuthenticationClient, AuthenticationClient>();
         builder.Services.AddSingleton<IResourceRegistryClient, ResourceRegistryClient>();
 
+        builder.Services.AddHttpClient<IAltinnRolesClient, AltinnRolesClient>()
+            .ReplaceResilienceHandler(static c =>
+            {
+                c.Retry.ShouldHandle = static _ => ValueTask.FromResult(false);
+                c.AttemptTimeout.Timeout = TimeSpan.FromSeconds(15);
```

Comment:
Får ikke økt timeout her til mer enn 15s uten å få runtime feil pga. total sample tid er 30s.
Får da runtime exception:
`Microsoft.Extensions.Options.OptionsValidationException: 'The sampling duration of circuit breaker strategy needs to be at least double of an attempt timeout strategy’s timeout interval, in order to be effective. Sampling Duration: 30s,Attempt Timeout: 30s'`

Hjalp heller ikke å prøve å sette `TotalRequestTimeout` til mer enn 30s

tips? @Alxandr